code/modules/customitems/item_defines.dm

```diff
@@ -533,6 +533,14 @@
 	icon_state = "hairflowerp"
 	item_state = "hairflowerp"
 
+/obj/item/clothing/head/valkyriehelmet //R3Valkyrie: Rikki
+	name = "charred visor"
+	desc = "A visor of alien origin, charred by fire and completely non-functioning. It's been impeccably polished, shiny!"
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "charred_visor"
+	species_restricted = list("Vox")
+	flags = BLOCKHAIR
```

Comment:
This mask's sprite (and the sprite it's based off) was designed to show Vox quills.